lib/crypto/test/crypto_SUITE.erl

```diff
@@ -1027,7 +1027,7 @@ group_config(rsa = Type, Config) ->
                   rsa_oaep(),
                   no_padding()
                  ],
-    Generate = [{rsa, 1024, 3}, {rsa, 2048, 17}, {rsa, 3072, 65537}],
+    Generate = [{rsa, 2048, 3}, {rsa, 3072, 65537}],
```

Comment:
For fun I just checked {1024,65537} and that works in fips mode.!  But {512,65537} gives expected error:
```
 {error,{openssl,[{"FIPS routines","RSA_BUILTIN_KEYGEN",
                                      "key too short"}]}}
```